sql/mob_spawn_slots.sql

```diff
@@ -0,0 +1,16 @@
+--
+-- Table structure for table `mob_spawn_slots`
+--
+
+/* mob spawn slots */
+DROP TABLE IF EXISTS `mob_spawn_slots`;
+CREATE TABLE `mob_spawn_slots` (
+  `id` int(10) unsigned NOT NULL,
+  `mobid` int(10) unsigned NOT NULL,
```

Comment:
reminder: per prior discussions in discord, we really need to tie it all up without relying on mobid being in 2 tables when this is done with. it'll mean having the "slot" id available somewhere else to tie them together. That way when mobIDs change we don't have to change it in multiple places.